### PR TITLE
Fix accessibility tests

### PR DESCRIPTION
**Pull request by cwrw**
1) The accessibility check should always be at the end of the step
   to not cause the steps to stop
2) We need to inject axe before each time we call it

This needs more cleanup and updating but this makes things pass
for now

### CODE REVIEW

File: cypress/integration/happy-path-with-accessibility.js [hunk 1]
```diff
@@ -34,20 +34,27 @@ function violationsCallBack(violations) {
 
 function checkAccessibility() {
   if(Cypress.config('accessibility')) {
+    cy.injectAxe();
```
Comment (tavy88) [inline cypress/integration/happy-path-with-accessibility.js]: The PR's description is not clear why was this needed and what this actually solves? Could you add these notes on please?

Comment (cwrw) [reply cypress/integration/happy-path-with-accessibility.js]: As suggested in the description and to re-iterate, we need to inject Axe as this step will fail otherwise each time we call it

Comment (tavy88) [reply cypress/integration/happy-path-with-accessibility.js]: The description just says `We need to inject axe before each time we call it` - it says what you did but not why you did it hence my question above. :)

Comment (cwrw) [reply cypress/integration/happy-path-with-accessibility.js]: Failure is implied if we need it
